Packs/ClarotyXDome/Integrations/XDome/XDome.py

```diff
@@ -19,9 +20,9 @@
 
 SMALLEST_TIME_UNIT = timedelta(seconds=1)
 
-MAX_REQUEST_LIMIT = 5000
+MAX_REQUEST_LIMIT = 5_000
 
-DEFAULT_REQUEST_LIMIT = 1000
+DEFAULT_REQUEST_LIMIT = 1_000
```

Comment:
why did you change the **DEFAULT_REQUEST_LIMIT** and **MAX_REQUEST_LIMIT**?

it might break backward compatibility for the users?

Reply:
Note that There's no changes to the actual values because python ignores the `_`. I was just trying to improve readability